src/redux/books/booksReducer.js

```diff
@@ -4,10 +4,13 @@ const initialState = {
   books: [],
 };
 
-export const addBook = (addedBook) => ({ type: types.ADD_BOOK, addedBook });
+export const addBook = (addedBook) => ({
+  type: types.ADD_BOOK,
+  payload: addedBook,
+});
 export const removeBook = (id) => ({
   type: types.REMOVE_BOOK,
-  id,
+  payload: id,
 });
```

Comment:
- [ ] Please display a few books from the Redux store when the page is rendered by using useSelector function. One of the goals is to display them in your React components.